File: twoSum.swift
func twoSum(_ nums: [Int], _ target: Int) -> [Int] {

    var range:Int = 0
    var dict:[Int:Int] = [:]
    
    for item in nums{
    let comp = target - item
        
        
        
    if dict[comp] != nil {
        let random:Int = dict[comp]!
        return [random, range]
    }else{
        dict[item] = range
    }
        
        
    range += 1
    }
    return [-1]

File: twoSum.cpp
vector<int> twoSum(vector<int>& nums, int target) {
        std::unordered_map<int,int>store;
       for(int i = 0; i < nums.size(); ++i){
           int comp = target - nums[i];
           if (store.find(comp) != store.end()) return {store[comp], i};
           else store[nums[i]] = i;
       } 
        return {-1,-1};
}
